tools/codegen/api/autograd.py

```diff
@@ -67,6 +67,14 @@ class ForwardDerivative:
     # Inputs for which the primal is required for this formula
     required_inputs_primal: Optional[Tuple[str, ...]]
 
+    # Flag to specify if this formula requires the original value of self
+    # This is only used by inplace operations
+    required_original_self_value: bool
+
+    # If this formula is specified by the user or if we are re-using the
+    # out of place formula for inplace
+    is_exact_match: bool
```

Comment:
Would prefer a better name on this. How about "reuses_outplace_formula"?

Reply:
Yes.
I re-used the one that was already in that function. But in this context it can be improved indeed.